pallets/xyk/src/tests/spot_price.rs

```diff
@@ -0,0 +1,83 @@
+use super::mock::*;
+use crate::XYKSpotPrice;
+use crate::*;
+use frame_support::assert_ok;
+use hydradx_traits::pools::SpotPriceProvider;
+use primitives::asset::AssetPair;
+use primitives::Price;
+
+#[test]
+fn spot_price_provider_should_return_correct_price_when_pool_exists() {
+	let asset_a = ACA;
+	let asset_b = DOT;
+
+	let initial = 99_000_000_000_000u128;
+
+	ExtBuilder::default()
+		.with_accounts(vec![(ALICE, asset_a, initial), (ALICE, asset_b, initial)])
```

Comment:
I am not sure. the ACA/DOT name does not really mean anything. it is all about the ids assigned to them

the main purpose to have asset_a/asset_b ids assigned before the use in the test is jus to make sure you always use the correct ids in the tests ( otherwise you would jus checking - do i use ACA or HDX or DOT when endowing some accounts?

and it also helps when you want to change the id ( and you rather use native asset instead of ACA).

Reply:
it is just seems more readable to me asset_a and asset_b is very similar as 80% is the same